tla/consistency/ExternalHistory.tla

```diff
@@ -37,11 +38,15 @@ TxIDs == Views \X SeqNums
 \* This models uses a dummy application where read-write transactions 
 \* append an integer to a list and then reads the list
 \* Read-only transactions simply read the list
+\* Transactions themselves simply consist of a natural number
 Txs == Nat
 
-\* History of events visible to clients
+\* History of events visible externally
+\* There is a single shared history for all nodes. The events themselves do not include timestamps.
+\* The order of events in the history is the order in which they were observed by a client (according to some global clock)
 VARIABLES history
 
+\* HistoryTypeOK is included here for reference but is often not checked in practice for performance reasons
```

Comment:
Consider removing this comment because it suggests `HistoryTypeOK` is not relevant.  However, it is a theorem of the spec, and would be critical for e.g. a proof.  The fact that we might consider too expensive for model checking should rather be a comment in the MC.tla or MC.cfg file.